net/mwan3/files/usr/sbin/mwan3track

```diff
@@ -322,11 +328,17 @@ main() {
 						fi
 					;;
 					nping-*)
-						WRAP nping -c $count $track_ip --${FAMILY#nping-} > $TRACK_OUTPUT &
+						WRAP nping -${FAMILY#ipv} -c $count $track_ip --${track_method#nping-} > $TRACK_OUTPUT &
```

Comment:
I don't like that. That must be moved to a separate commit with a description of what you are fixing there

Reply:
Do you mean separate **commit** or seperate/new PR?

Reply:
You could add this to this PR. But in a separate commit.

Reply:
> You could add this to this PR. But in a separate commit.

Done.